Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -190,7 +198,9 @@ public function processRequest(ActionRequest $request, ActionResponse $response)
         $this->actionMethodName = $this->resolveActionMethodName();
 
         $this->initializeActionMethodArguments();
-        $this->initializeActionMethodValidators();
+        if ($this->enableDynamicTypeValidation !== true) {
```

Comment:
nitpick: any specific reason why you didn't go for
```suggestion
        if (!$this->enableDynamicTypeValidation) {
```

Reply:
Nope, just accustomed to strict boolean checks over the falsy operator.